packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -0,0 +1,311 @@
+import * as React from 'react';
+import Frame, { FrameContextConsumer } from 'react-frame-component';
+
+import { DebugSelector, FiberNavigator, Provider, teamsTheme } from '@fluentui/react-northstar';
+import { JSONTreeElement } from './types';
+import { EventListener } from '@fluentui/react-component-event-listener';
+import { fiberNavFindJSONTreeElement, fiberNavFindOwnerInJSONTree, renderJSONTreeToJSXElement } from '../config';
+import { DebugFrame } from './DebugFrame';
+import { DropSelector } from './DropSelector';
+
+const Canvas = ({
+  draggingElement,
+  isExpanding,
+  isSelecting,
+  jsonTree,
+  onDropPositionChange,
+  onMouseMove,
+  onMouseUp,
+  onSelectComponent,
+  onSelectorHover,
+  selectedComponent,
+  onCloneComponent,
+  onMoveComponent,
+  onDeleteComponent,
+  onGoToParentComponent,
+  renderJSONTreeElement,
+  style,
+}: {
+  draggingElement: JSONTreeElement;
+  jsonTree: JSONTreeElement;
+  isExpanding?: boolean;
+  isSelecting?: boolean;
+  onDropPositionChange: (dropParent: JSONTreeElement, dropIndex: number) => void;
+  onMouseMove?: ({ clientX, clientY }: { clientX: number; clientY: number }) => void;
```

Comment:
```suggestion
  onMouseMove?: (e: React.MouseEvent<HTMLDivElement>) => void;
```

Please also other typings and use proper event types

Reply:
This is not the whole event, just x and y